index.html

```diff
@@ -16,39 +16,48 @@
       <!-- <h1>Saul Greene</h1> -->
       <span class="icon-menu"></span>
       <ul>
-        <li><a href="#">ABOUT</a></li>
-        <li><a href="#">PROJECTS</a></li>
-        <li><a href="#">RESUME</a></li>
-        <li><a href="#">CONTACT</a></li>
+        <li class="nav" data-content= "about"><a href="#">ABOUT</a></li>
```

Comment:
**TODO FOR LOST POINTS**

Something funky is going on with here, probably with your CSS, that I'm unable to access your nav bar at all until I shrink my screen to less than 885 px, and I had to hunt through the console HTML to even find your nav bar. 

And even at that size, your hamburger menu is still very small and hardly noticeable. So UX-wise, it's a little rough, even though when I figured things out, I can see you have your functionality working properly.